plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -25,6 +25,39 @@ class WC_Helper_Admin {
 	 */
 	public static function load() {
 		add_filter( 'woocommerce_admin_shared_settings', array( __CLASS__, 'add_marketplace_settings' ) );
+		add_filter( 'rest_api_init', array( __CLASS__, 'register_rest_routes' ) );
+	}
+
+	/**
+	 * Registers the REST routes for the Helper.
+	 *
+	 * @return void
+	 */
+	public static function register_rest_routes() {
+		register_rest_route( 'wc/v1', '/marketplace/featured', array(
```

Comment:
The official WooCommerce Rest API is currently at `v3` and uses the prefix `wc/v3`. This endpoint, however, is the first iteration. So I didn't know which prefix to choose. This endpoint is also for only Marketplace purposes.

Reply:
Not sure what you'd do here either to be honest. I guess, by using v1, we're providing a fallback that is backwards compatible and it still won't conflict since this is new 🤷🏻

Reply:
This is a real conundrum! Hopefully one of the core testers will highlight if this is wrong.

Reply:
Reading the documentation, I think new endpoints should also be `v3`. When WooCommerce Rest API bumps the version number, all endpoints are updated to the latest version number. The old endpoints are archived. `v3` is the current version, `v2` and `v1` are archived. So it feels wrong to use `v1`. 

https://github.com/woocommerce/woocommerce/wiki/Contributing-to-the-WooCommerce-REST-API#api-versioning-and-how-it-affects-development